src/plotman/analyzer.py

```diff
@@ -17,21 +17,30 @@ def analyze(logfilenames, bytmp, bybitfield):
             phase_time = {}  # Map from phase index to time
             n_sorts = 0
             n_uniform = 0
+            is_first_last = False
 
             # Read the logfile, triggering various behaviors on various
             # regex matches.
             for line in f:
                 # Beginning of plot job.  We may encounter this multiple
                 # times, if a job was run with -n > 1.  Sample log line:
-                # Starting plotting progress into temporary dirs: /mnt/tmp/01 and /mnt/tmp/a
-                m = re.search(r'^Starting plotting.*dirs: (.*) and (.*)', line)
+                # 2021-04-08T13:33:43.542  chia.plotting.create_plots       : INFO     Starting plot 1/5
```

Comment:
I guess there's been some log format changes?  Do you know what chia-blockchain version?

Reply:
Not quite, it's more a confusing diff.  Behavior is essentially the same as before (the previous regex can still be seen below, in the next matcher), we're just also matching on an additional earlier line in the log, and we've moved some of the data-initialization here because it's now the first line that we match.